scss/components/tile.scss

```diff
@@ -79,6 +81,7 @@ $block: #{$fd-namespace}-tile;
     @include fd-reset-spacing();
```

Comment:
is this still a needed mixin()?

Reply:
Between spacing, type, color and it's other includes, it covered everything inside fd-reset already.